src/main.rs

```diff
@@ -132,16 +132,14 @@ fn main() -> Result<()> {
         let (v, command) = submatches.unwrap().subcommand();
         let verb = Trust_subcommands::from_str(v);
         let app_id = arguments::get_app_id(&command.unwrap(), &context)?;
+        let days: i64 = match command.unwrap().value_of(&Parameters::days) {
+            Some(d) => d.parse().unwrap(),
+            _ => trust::CERT_VALIDITY_DAYS,
+        };
```

Comment:
I would pass down an `Option` here and do default thing in `trust.rs`

Reply:
Okay :+1: